### PR TITLE
Loyalty case study

### PR DESCRIPTION
**Pull request by charypar**
#### Description

Starts a page with the Nando's Loyalty case study. Missing all graphics for now

- [x] Move text over from google doc
- [x] Set up basic structure
- [ ] Add Ryan photo
- [ ] Add visuals as per design

##### TIER 1

_Full support. Test all front-end PR's against these._

_WIN10_

- [ ] Chrome
- [ ] Firefox
- [ ] Edge

_Mac High Sierra_

- [ ] Chrome
- [ ] Safari

_iOS: iPhone_

- [ ] Last 2 major iOS versions

_iOS: iPad_

- [ ] Last 2 major iOS versions

_Android_

- [ ] Last 3 major Android versions; check with mixture of devices

##### TIER 2

_Functional support; some visual differences allowed. Only test these if it's mentioned in the test plan._

_WIN 7_

- [ ] IE11
- [ ] Chrome

_Mac Sierra_

- [ ] Safari

_iOS: iPhone_

- [ ] +1 major iOS versions compared to Tier 1

_iOS: iPad_

- [ ] +1 major iOS versions compared to Tier 1

_Ubuntu_

- [ ] Firefox
- [ ] Chromium

### CODE REVIEW

File: site/pages/our-work/case-study/nandos-loyalty/index.js [hunk 1]
```diff
@@ -0,0 +1,379 @@
...
+import RyanX2Img from './images/Ryan@2x.png';
+import TapWithPhoneX1Img from './images/TapWithPhone@1x.png';
+import TapWithPhoneX2Img from './images/TapWithPhone@2x.png';
+import TapWithWatchX1Img from './images/TapWithWatch@1x.png';
+import TapWithWatchX2Img from './images/TapWithWatch@2x.png';
+import TestingX1Img from './images/Testing@1x.png';
+import TestingX2Img from './images/Testing@2x.png';
+
+import styles from './style.css';
+// import Triptych from './triptych';
```
Comment (charypar) [inline site/pages/our-work/case-study/nandos-loyalty/index.js:43]: I was aiming to re-enable this code for use later, we should get rid of it if we're not using it.

File: site/pages/our-work/case-study/nandos-loyalty/index.js [hunk 2]
```diff
@@ -0,0 +1,379 @@
...
+//   srcImageRight: right,
+//   altImageLeft:
+//     "Phone with the Pride app on in screen displaying a map, text reads: 'Easily find bars and events on the day'",
+//   altImageMiddle:
+//     "Phone with the Pride app on in screen displaying a list of events, text reads: 'Plan your perfect festival with ease and love'",
+//   altImageRight:
+//     "Phone with the Pride app on in screen displaying a list of event categories, text reads: 'Find diverse experiences from cabaret to films'",
+// };
+
+export default function Pride() {
```
Comment (charypar) [inline site/pages/our-work/case-study/nandos-loyalty/index.js:99]: My mistake, should be called `NandosLoyalty`

File: site/pages/our-work/case-study/nandos-loyalty/index.js [hunk 3]
```diff
@@ -0,0 +1,379 @@
...
+          className={styles.headerImg}
+          altText="People (Badgers) sticking post-its on a blue wall"
+        />
+      </div>
+      <Body>
+        <PageHeading>Digital-first loyalty programme</PageHeading>
+        <Section>
+          <SectionBody>
+            <P>
+              <a href="https://www.nandos.co.uk/card/about">Nando&:aposs loyalty programme</a>{' '}
```
Comment (charypar) [inline site/pages/our-work/case-study/nandos-loyalty/index.js:119]: Also my bad, clearly can't type
```suggestion
              <a href="https://www.nandos.co.uk/card/about">Nando&apos;s loyalty programme</a>{' '}
```

File: site/pages/our-work/case-study/nandos-loyalty/index.js [hunk 4]
```diff
@@ -0,0 +1,379 @@
...
+                to their free PERi-PERi. On top of that, the scheme released more plastic into the
+                world, which wasn’t aligned with Nando’s environmental efforts such as their
+                commitment to reduce direct emissions to zero by 2030.
+              </P>
+              <P>The plastic card had to evolve with the times.</P>
+            </div>
+          </Section>
+        </div>
+        {
+          // <Triptych {...tryptychConfig} />
```
Comment (charypar) [inline site/pages/our-work/case-study/nandos-loyalty/index.js:211]: We should remove this guy if no longer used.

**Review by charypar — COMMENTED** [review]
Leaving some notes for before we ship this

File: site/pages/our-work/case-study/nandos-loyalty/index.js (hunk 3 above)
Comment (SiAshbery) [reply site/pages/our-work/case-study/nandos-loyalty/index.js]: Think I fat thumbed a brand new mistake on this one 😅

**Pull request merged**